.docs/02-getting-started.md

```diff
@@ -0,0 +1,11 @@
+[← Introduction][back] - [:book:][index] - [Adding Structure →][next]
+
+# Getting Started
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur eget commodo magna. Nam varius, purus tincidunt pulvinar auctor, sapien lorem consequat tortor, in cursus enim sapien vel erat. Donec non sollicitudin neque. Integer rutrum egestas purus ac porttitor. Morbi maximus metus dapibus iaculis molestie. Nam euismod sed est quis auctor. Aenean at odio vitae orci sodales lobortis nec a nibh. Maecenas condimentum ipsum risus, eu pellentesque augue facilisis a. In hac habitasse platea dictumst. Quisque fermentum accumsan felis, eu venenatis nisl pulvinar vel. Nam eu accumsan est. Sed viverra vel ligula sit amet placerat. Donec molestie ut mauris nec egestas.
```

Comment:
It might make sense to separate "Getting Started with API" from "Getting Started with Config". Not sure which one to lead with. The README previously covered getting started with API.